spec/unit/vault/rails_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+RSpec.describe Vault::Rails do
+  describe "#memory_key_for" do
+    input_examples = [
+      ["path", "key"],
+      ["path", "key", "context"],
+      ["a_really_long_path", "a_really_long_key"],
+      ["a_really_long_path", "a_really_long_key", "a_really_long_context"],
+    ]
+
+    input_examples.each do |path, key, encryption_context|
+      context "with path=#{path}, key=#{key}, context=#{encryption_context}" do
+        it "returns exactly 16 bytes as required by OpenSSL AES 128" do
+          memory_key = Vault::Rails.send(
+            :memory_key_for, "foo", key, context: encryption_context
```

Comment:
What is the purpose of `"foo"` vs the `path` variable?

Reply:
Thanks! This was a typo 😬

Reply:
Fixed!

Reply:
Ah! ok then do you mind updating L26 test too?